pages/common/views/components/pil.jsx

```diff
@@ -0,0 +1,82 @@
+import React, { Component } from 'react';
```

Comment:
This file isn't currently rendered anywhere in the current implementation but will be needed later once we render different content depending on task type. 

Do you want me to take it out, or leave it in for whoever picks up the ticket to do that work?